docroot/.htaccess

```diff
@@ -69,6 +69,20 @@ AddEncoding gzip svgz
 <IfModule mod_rewrite.c>
   RewriteEngine on
 
+  # Forbid access to install and update files.
+  RewriteRule ^(core/install|update)\.php - [F]
```

Comment:
This shouldn't happen locally. We can just check to make sure we're on cloud with the same method you used below:
```
RewriteCond %{ENV:AH_SITE_ENVIRONMENT} ^.+$
```